cli/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:20-alpine3.18 as core
```

Comment:
I think the latest version is alpine 3.19

Reply:
Thanks, will fix